world/config/skillmanager.py

```diff
@@ -0,0 +1,69 @@
+from world.abilities.ability import Ability
+
+#Create the skills database
+skill_manager = {}
+
+# hide skill
+skill_manager['hide'] = Ability()
+skill_manager['hide'].name = 'hide'
```

Comment:
I'm not to sure how to go about this.

If I create an instance of Ability and assign it as a value within a dictionary like this: 
skill_manager = {
                             'hide' : Ability(),
                             'kick' : Ability(),
                           }

How do I go about modify specific attributes using a dictionary literal format, i.e. what would the equivalent of this be:

skill_manager['hide'].level = 100